[ 시세 그래프 응답 DTO] datapoints에 상품 id와 title까지 함께 전달

yunjeooong opened the pull request:
Title
[ 시세 그래프 응답 DTO] datapoints에 상품 id와 title까지 함께 전달

## Description
- 프론트 담당자분 요청으로  시세 그래프 응답 DTO 내부 datapoints에 상품 id와 title까지 함께 전달하도록 추가하였습니다.
- Swagger에서 JSON 형태로 출력되던 문제를 해결하고, **블록 형태(Query Parameter 방식)**로 보이도록 수정하였습니다. 

## Changes
- datapoints에 상품 id와 title이 추가됐습니다. (class PriceGraphResponseDTO)
- Query Parameter로 페이징 처리:
기존 Pageable 객체를 사용하지 않고, page, size, sort를 개별 Query Parameter로 분리 처리.@RequestParam으로 page(페이지 번호), size(페이지 크기), sort(정렬 조건)를 수동으로 받아 Pageable 객체를 직접 생성했습니다.

## Screenshots
![image](https://github.com/user-attachments/assets/a51b5056-eadb-4e01-b042-e43d753a5e45)


## Ref

yunjeooong committed:
feat(dto): datapoints에 상품 id와 title까지 함께전달

nagul2 commented:
스웨거 try it out시 제이슨 형태가 아니라 블럭 형태로 나와야 하는데, 확인한번 부탁드립니다.
![image](https://github.com/user-attachments/assets/3b86ccdd-95b6-4d91-9df3-6eea7f654764)
요런식으로 스웨거에 보이도록 통일해서 반환해야 합니당

yunjeooong committed:
refactor: Jackson 관련 어노테이션 모두 제거

yunjeooong committed:
feat: API 요청 파라미터 형식 개선

- @ModelAttribute를 @RequestParam으로 변경하여 개별 파라미터 처리
- 컨트롤러에서 DTO 수동 생성 방식으로 변경
- @RequestParam에 기본값 설정으로 선택적 파라미터 지원
- Swagger UI에서 블록 형태로 파라미터 표시되도록 개선

yunjeooong committed:
feat: Swagger에서 Query Parameters 블록 형태로 표시 및 Pageable 수동 처리
